src/ResponseConfigurator/ConfigurableResponseCacheConfigurator.php

```diff
@@ -32,11 +33,14 @@ class ConfigurableResponseCacheConfigurator implements ResponseCacheConfigurator
      */
     private $defaultTtl;
 
-    public function __construct($enableViewCache, $enableTtlCache, $defaultTtl)
+    private $tagHandler;
```

Comment:
Nitpick: please add a type hint in PhpDoc

Reply:
fixed